README.md

```diff
@@ -61,8 +61,16 @@ Visiting `http://localhost:3000` will allow you to `Modify scenarios`. The defau
 
 ## API
 
+### initialise
+
+returns the internal express instance
+
+> `function({ default, scenarios, options })`
+
```

Comment:
Can we make this `createExpressApp` rather than `initialise`? In this case I'd like it to be clear.

And then capitalise `returns` to `Returns`.